homeassistant/components/epson/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Epson Projector",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "name": "Name",
+          "port": "[%key:common::config_flow::data::port%]",
+          "ssl": "Use SSL (not tested)"
```

Comment:
Not sure if this is a nice way of presenting the user with an option. Perhaps someone else can comment on that

Reply:
Maybe I will explain this option little bit further. Epson Android app has option to enable encrypted communication, but through SSL, but my projector is rejecting to enable encrypted communication.
I know that some users are using it, but I don't know how to verify if it's working.

Reply:
I would agree that this isn't the place. Possibly better as a note in the documentation that its known to cause issues.